pyproject.toml

```diff
@@ -69,8 +69,8 @@ pylint = "^3.1.0"
 matplotlib = "^3.7.0"
 tensorflow = { version = "^2.16.1", markers = "sys_platform == 'linux'" }
 torch = "^2.1.1"
-qibojit = { git = "https://github.com/qiboteam/qibojit.git", branch="clifford_simulator_numba" }
-qibotn = { git = "https://github.com/qiboteam/qibotn.git" }
+qibojit = { git = "https://github.com/qiboteam/qibojit.git", branch="list_backends" }
```

Comment:
I think https://github.com/qiboteam/qibojit/pull/172 can be merged. So maybe we should merge the PR in qibojit first and point this back to main before merging?